challenge_scoring/metrics/valid_connections.py

```diff
@@ -4,93 +4,135 @@
 from __future__ import division
 
 import logging
-from itertools import chain
 
+from dipy.io.stateful_tractogram import StatefulTractogram
 from dipy.segment.clustering import QuickBundles
 from dipy.segment.metric import AveragePointwiseEuclideanMetric
 from dipy.tracking.distances import bundles_distances_mdf
 from dipy.tracking.streamline import set_number_of_points
-from nibabel.streamlines import Tractogram
 import numpy as np
 
 from challenge_scoring import NB_POINTS_RESAMPLE
-from challenge_scoring.metrics.bundle_coverage import compute_bundle_coverage_scores
+from challenge_scoring.metrics.bundle_coverage import \
+    compute_bundle_coverage_scores
 
+# todo. See if this could be bigger nowadays.
+CHUNK_SIZE = 10000
```

Comment:
It does work on my computer but it does not give the same results as in 2015 so for reproducibility I suggest we switch back to 5000.